.github/workflows/linters.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: Linters
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - stable-*
+    tags:
+      - '*'
+
+jobs:
+  linters:
+    uses: ansible-network/github_actions/.github/workflows/tox-linters.yml@main
```

Comment:
@hakbailey I've incorporated the isort command. Addressing the errors thrown by mypy will require more effort, and I propose tackling it post-release. Additionally, since we need to implement it across all our repositories, what are your thoughts on taking the task after the release?

Reply:
I think that's probably fine. Just out of curiosity, does it still give a lot of errors if you do something similar to what's already implemented in the [cloud.terraform collection](https://github.com/ansible-collections/cloud.terraform/blob/8c52bf2a0a0f5e3ba56f57d0ac5bf1dd8efd0788/tox.ini#L32)?

Reply:
Yes, there are definitely ways we can configure mypy to limit the amount of upfront work that's needed, but we should limit this PR to fixing only what's necessary right now. Our goal is just to get the CI to a state that we can actually merge things. We can focus on improving it later.